db/migrate/20201126054738_add_index_to_handle.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToHandle < ActiveRecord::Migration[6.1]
+  def change
+    add_index :users, :handle, unique: true
+  end
+end
```

Comment:
Could you merge all these migrations together please. I'm keeping it to one migration per model to make the ETL easier later.